doc/devel/triage.rst

```diff
@@ -0,0 +1,214 @@
+
+.. _bug_triaging:
+
+Bug triaging and issue curation
+================================
+
+The `issue tracker <https://github.com/matplotlib/matplotlib/issues>`_
+is important to the communication in the project: it is where users to
+make feature requests and report bugs, developers identify major
+projects to work on and discuss priorities. For this reason, it is
+important to curate the issue list, adding labels to issues and
+closing issues that are not necessary.
+
+
+Triaging issues does not require any particular expertise in the
+internals of Matplotlib, is extremely valuable to the project, and we
+welcome anyone to participate in issue triage!  However, people who
+are not part of the Matplotlib organization do not have `permissions
+to change milestones, add labels, or close issue
+<https://docs.github.com/en/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization>`_.
+If you do not have enough permissions do to a step, please leave a
+comment tagging ``@matplotlib/triage_team`` with your recommendations.
+
+Working on issues to improve them
+---------------------------------
+
+Improving issues increases their chances of being successfully resolved.
+Guidelines on submitting good issues can be found :ref:`here
+<submitting-a-bug-report>`.
+A third party can give useful feedback or even add
+comments on the issue.
+The following actions are typically useful:
+
+  - documenting issues that are missing elements to reproduce the problem
+    such as code samples
+
+  - suggesting better use of code formatting
```

Comment:
link to code format style here?

Reply:
I think this is aimed at improving mark-down usage not the Python, clarified.